### ISSUE TITLE
[question] What unit should altitude be?

### ISSUE THREAD

**Issue by mjmadsen** [tejado/pokemongo-api-demo#224]
What unit should altitude be?

**Comment by dragon-jlvl** [comment]
yo use GD, de esta pagina..... 

http://www.coordenadas-gps.com/

**Comment by dnsBlah** [comment]
indeed, just to add
put some randomness as well...

**Comment by mjmadsen** [comment]
Thanks guys.

**Comment by dnsBlah** [comment]
```
import geocoder
g = geocoder.elevation('<address or [lat,lng]>')
g.meters
```

should do trick as well most of the times

**Comment by dnsBlah** [comment]
oops, with address as string or list of lat,lon

**Comment by mjmadsen** [comment]
@dnsBlah Meters then?

**Comment by dnsBlah** [comment]
@mjmadsen sorry, see updated comment.
I was on iphone before. I took this from a sample I found few weeks earlier.
Have not got time to implement it myself. Also my google directions is still pending

g.meters is the altitude in meters :)
p.s. this function might not ALWAYS give response.. There is also a google elevations API tho, which is more promising: [https://developers.google.com/maps/documentation/elevation/intro](https://developers.google.com/maps/documentation/elevation/intro)
Also look at directions, ;-)
here is a sample I'm working on

```
import googlemaps

mapService = googlemaps.Client("googleKey")

navDirections = mapService.directions("From address or lat,lon",""To address or lat,lon", mode='walking')
directions = []

for step in navDirections[0]['legs'][0]['steps']:
    tLocation = step['end_location']
    waypoint = [tLocation['lat'],tLocation['lng']]
    directions.append(waypoint)
```

This will give you 'waypoints' from one point to another ,like navigation.
However if you follow the waypoints its not going 'around' corners, but straight from one waypoint to another.
**At least its better than from point A to B in a straight line...
